gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -247,10 +257,21 @@ public LatLngBounds getViewport() {
   }
 
   @Override
-  public void renderTileOverlay() {
-    try (MapBoxOfflineTileProvider tileProvider =
-        new MapBoxOfflineTileProvider(context.getFilesDir())) {
-      map.addTileOverlay(new TileOverlayOptions().tileProvider(tileProvider));
-    }
+  public void renderTileOverlays(ImmutableSet<String> mbtilesFiles) {
```

Comment:
Same here: `add` instead of `render`

Reply:
57cb7dd9d31e68d8c97168a3778b7bcb62c173f7